presto-delta/src/main/java/com/facebook/presto/delta/DeltaSessionProperties.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.delta;
+
+import com.facebook.presto.cache.CacheConfig;
+import com.facebook.presto.hive.HiveClientConfig;
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.session.PropertyMetadata;
+import com.google.common.collect.ImmutableList;
+import io.airlift.units.DataSize;
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static com.facebook.presto.hive.HiveSessionProperties.dataSizeSessionProperty;
+import static com.facebook.presto.spi.session.PropertyMetadata.booleanProperty;
+
+public final class DeltaSessionProperties
+{
+    public static final String PARTITION_PRUNING_ENABLED = "partition_pruning_enabled";
```

Comment:
There is no getter/setter for this property. If you really want to guard the partition pruning by this session property, shall we create the getter and setter methods? 
I'd actually suggest removing this session property and config property. Partition pruning was on by default in Hive, and it'll be adding to the management labor to the users, unless it's really a risky operation.

Reply:
This property is removed now.